CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+- All redundancy finder methods will return four items --- redundant groups, bin centers, lengths, and conjugates.
+- UVData.get_antenna_redundancies will no longer automatically conjugate baselines.
+- UVData.get_baseline_redundancies and UVData.get_antenna_redundancies have been combined.
+
```

Comment:
should add a deprecated section to call out the deprecated methods.

Reply:
Done